src/Illuminate/Mail/TransportManager.php

```diff
@@ -70,14 +71,13 @@ protected function createSesDriver()
 
         $config += [
             'version' => 'latest',
-            'service' => 'email',
-            'credentials' => [
-                'key'    => $config['key'],
-                'secret' => $config['secret'],
-            ],
+            'service' => 'email', // Fixes issue with SDK versions < 3.0.2
```

Comment:
Do we care about 3.0.0 and 3.0.1? I think we should just say we support `^3.0.2`?